src/registrar/assets/sass/_theme/_admin.scss

```diff
@@ -127,11 +135,20 @@ html[data-theme="dark"] {
     .change-list .usa-table--borderless thead th,
     .change-list .usa-table thead td, 
     .change-list .usa-table thead th,
+    .change-form .usa-table,
+    .change-form .usa-table--striped tbody tr:nth-child(odd) td,
+    .change-form .usa-table--borderless thead th,
+    .change-form .usa-table thead td, 
+    .change-form .usa-table thead th,
     body.dashboard,
     body.change-list,
-    body.change-form {
+    body.change-form,
+    .analytics {
```

Comment:
Correct. In my opinion, CSS selectors should be as general as you can get away with for performance, as specific as they need to be to avoid conflicts.

Reply:
Rule of thumb is no more than 3 deep btw